app/assets/stylesheets/govuk_publishing_components/components/_attachment.scss

```diff
@@ -0,0 +1,43 @@
+$thumbnail-width: 99px;
+$thumbnail-height: 140px;
+$thumbnail-border: 5px;
+$thumbnail-border-colour: rgba(11, 12, 12, .1);
+$thumbnail-shadow-colour: rgba(11, 12, 12, .4);
+
+.gem-c-attachment {
+  @include govuk-font(19);
+  @include govuk-clearfix;
+}
+
+.gem-c-attachment__thumbnail {
+  float: left;
+  margin-bottom: govuk-spacing(3);
+  margin-right: govuk-spacing(5);
+  padding: $thumbnail-border;
+  position: relative;
+}
+
+.gem-c-attachment__thumbnail-image {
+  display: block;
+  width: $thumbnail-width;
+  max-width: 100%;
+  height: $thumbnail-height;
+  outline: $thumbnail-border solid $thumbnail-border-colour;
+  background: govuk-colour("white");
+  box-shadow: 0 2px 2px $thumbnail-shadow-colour;
```

Comment:
Stylistic comment: can we move the box shadow width into its own variable at the top.
I'm aware I introduces this line in Content Publisher, but it's good practice to have every value in a variable and we don't have yet the linting setup properly.

Reply:
I've called this `$thumbnail-shadow-width`.